File: Assignments/Assignment 33/C Codes/program33_4.c
// ACCEPT CHARACTER FORM USER AND CHECK IT IS SMALL OR NOT.

// INPUT - g        // OUTPUT - TRUE
// INPUT - D        // OUTPUT - FALSE

#include<stdio.h>

#define TRUE 1
#define FALSE 0

typedef int BOOL; 

BOOL ChkDigit(char ch)
{
    if((ch >= 'a' && ch <= 'z'))
    {
        return TRUE;
    }
    else
    {
        return FALSE;
    }
}

int main()
{
    char cValue = '\0';
    BOOL bRet = FALSE;

    printf("Enter Character : \n");
    scanf("%c", &cValue);

    bRet = ChkDigit(cValue);

    if(bRet == TRUE)
    {
        printf("It is Small Character\n");
    }
    else
    {
        printf("It is not a Small Character\n");
    }

    return 0;
}

File: Assignments/Assignment 33/Java Codes/program33_4.java
// ACCEPT CHARACTER FORM USER AND CHECK IT IS SMALL OR NOT.

// INPUT - g        // OUTPUT - TRUE
// INPUT - D        // OUTPUT - FALSE


import java.util.*;

class CheckX
{
    boolean ChkSmall(char ch)
    {
        if((ch >= 'a' && ch <= 'z'))
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class program33_4
{
    public static void main(String A[]) 
    {
        char cValue = '\0';
        boolean bRet = false;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Character : ");

        cValue = sobj.nextLine().charAt(0);       // Important

        CheckX cobj = new CheckX();

        bRet = cobj.ChkSmall(cValue);

        if(bRet == true)
        {
            System.out.println("It is Small Character\n");
        }
        else
        {
            System.out.println("It is not Small Character\n");
        }
    }
}